tools/common/Frameworks.cs

```diff
@@ -484,6 +486,8 @@ public static Frameworks GetwatchOSFrameworks (bool is_simulator_build)
 
 				{ "Accessibility", "Accessibility", 7,0 },
 				{ "UniformTypeIdentifiers", "UniformTypeIdentifiers", 7,0 },
+
+				{ "CoreLocationUI", "CoreLocationUI", 8,0 },
```

Comment:
why ? it does not look like it's available for watchOS

Reply:
I think we can remove it and add the ignore